packages/react/src/components/cookieConsent/CookieConsent.module.scss

```diff
@@ -0,0 +1,278 @@
+.container {
+  position: fixed;
+  left: 0;
+  width: 100vw;
+  z-index: 999;
+  bottom: 0;
+  --common-spacing: var(--spacing-s);
+}
+
+.aligner {
+  position: absolute;
+  bottom: 0;
+  z-index: 2;
+  width: 100%;
+  overflow-y: scroll;
+  max-height: 100vh;
+}
+
+.container .aligner {
+  transform: translateY(100%);
+  transition: transform 1s;
+}
+
+.container.animate-in .aligner {
+  transform: translateY(0%);
+}
+
+.buttons {
+  display: flex;
+  flex-direction: column;
+  width: 100%;
+  padding-top: var(--spacing-l);
+}
+
+.buttons > * {
+  margin: 0 var(--common-spacing) var(--common-spacing) 0;
+  width: 100%;
+}
+
+.content {
+  position: relative;
+  padding: var(--common-spacing) var(--common-spacing) 0 var(--common-spacing);
+  background: #ffffff;
+  width: 100%;
+  box-sizing: border-box;
+  border-top: 8px solid var(--color-bus);
+}
+
+.language-switcher {
+  position: absolute;
+  top: var(--common-spacing);
+  left: var(--common-spacing);
+}
+
+.language-selector-override {
+  position: relative;
+  right: unset;
+}
+
+.language-selector-override > button {
+  padding-top: 0;
+}
+
+.language-selector-override > button + div {
+  right: unset;
+  left: 0;
+}
+
+.main-content,
+.text-content {
+  padding: 0;
+}
+
+.text-content > p {
+  padding-bottom: var(--spacing-2-xl);
+}
+
+.main-content {
+  padding-top: calc(var(--common-spacing) * 2);
```

Comment:
It might be more simple to just use hds design token variable `--spacing-l` here.